test/snapshots/ansible-copy-nochange.json.snapshot.json

```diff
@@ -49,9 +49,7 @@
             "secontext": "unconfined_u:object_r:user_home_t:s0",
             "size": 446,
             "state": "file",
-            "uid": 1001,
-            "ansible_facts": {
-            }
+            "uid": 1001
```

Comment:
This is good, delete this sub-hash from the report result this is good.

I would ask you to update snapshot tests to also create two instead on JSON files: one for report, one with facts. You can change the naming convention to something like `ansible-xyz.report.json.snapshot.json` and `ansible-xyz.facts.json.snapshot.json`